dp/cloud/docker/python/protocol_controller/Dockerfile

```diff
@@ -1,22 +1,42 @@
 ARG ENV=standard
-FROM python:3.9.2-slim-buster as standard-version
+FROM python:3.9.2-slim-buster as protos-generator
+
+RUN apt update && apt install -y curl zip make
+RUN curl -Lfs https://github.com/protocolbuffers/protobuf/releases/download/v3.18.0/protoc-3.18.0-linux-x86_64.zip \
+    -o protoc3.zip
+RUN unzip protoc3.zip -d protoc3 &&\
+    mv protoc3/bin/protoc /bin/protoc &&\
+    chmod a+rx /bin/protoc &&\
+    mkdir -p /magma &&\
+    mv ./protoc3/include/google /magma/google &&\
+    rm -rf protoc3.zip protoc3
+RUN pip3 install protobuf setuptools==49.6.0 grpcio==1.37.1 grpcio-tools==1.37.1
+ENV MAGMA_ROOT=/magma
```

Comment:
might as well declare it a few lines above and use it instead of /magma

Reply:
fixed